frontend/deploy-smoke.js

```diff
@@ -41,5 +41,6 @@ driver
       return;
     }
     console.log("Smoke test encountered unknown failure.");
+    console.log(`Root element value was: ${value}`);
```

Comment:
This is added to help provide additional context if we have another event of the smoke test being flaky